src/utils/SiriusAdapter.cpp

```diff
@@ -289,20 +254,36 @@ class TOPPSiriusAdapter :
     //Convert sirius_output to mztab and store file
     MzTab sirius_result;
     MzTabFile siriusfile;
-    SiriusMzTabWriter::read(subdirs, number, sirius_result);
-    siriusfile.store(out1, sirius_result);
+    SiriusMzTabWriter::read(subdirs, number_compounds, sirius_result);
+    siriusfile.store(out_sirius, sirius_result);
 
     //Convert sirius_output to mztab and store file
-    if (!out2.empty() && fingerid == true)
+    if (!out_csifingerid.empty() && fingerid)
     {
       MzTab csi_result;
       MzTabFile csifile;
-      CsiFingerIdMzTabWriter::read(subdirs, number, csi_result);
-      csifile.store(out2, csi_result);
+      CsiFingerIdMzTabWriter::read(subdirs, number_compounds, csi_result);
+      csifile.store(out_csifingerid, csi_result);
     }
 
-    //clean tmp directory
-    removeTempFiles_(tmp_dir, tmp_ms_file);
```

Comment:
Probably just a matter of taste. Function is never going to called anywhere else, so I inlined the code.